package.json

```diff
@@ -26,6 +27,7 @@
     "eslint": "^6.5.1",
     "jest": "^24.9.0",
     "metro-react-native-babel-preset": "^0.56.0",
+    "react-native-svg-transformer": "^0.14.3",
```

Comment:
Do we need SVG transformer?

Reply:
To use svg from local its needed.

Reply:
React-Native-SVG does the job. according to thier docs we will use transformer when we want to change the SVG in run time.
https://github.com/react-native-community/react-native-svg#use-with-svg-files

Reply:
I have implemented it @sujith22  with React-Native-SVG